src/util.c

```diff
@@ -131,6 +131,17 @@ static unsigned long id_or_max(const char *name)
 	return id;
 }
 
+bool hyper_is_id(const char *name, unsigned long *val)
+{
+	char *ptr;
+	errno = 0;
+	long id = strtol(name, &ptr, 10);
+	if (name == ptr || id < 0 || (errno != 0 && id == 0) || *ptr != '\0')
```

Comment:
Just following hyperstart coding standards here. I have been asked to remove the braces for earlier commits by them.